input/postgres/relation_stats_ext.go

```diff
@@ -0,0 +1,96 @@
+package postgres
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/lib/pq"
+	"github.com/pganalyze/collector/state"
+	"github.com/pganalyze/collector/util"
+)
+
+const extendedStatisticsSQLInheritedField = "null"
+const extendedStatisticsSQLpg15InheritedField = "se ->> 'inherited'"
+
+// pg_get_statisticsobjdef_expressions(s.oid) provides pg_stats_ext.exprs without accessing pg_stats_ext
+const extendedStatisticsSQL = `
+WITH relation_stats_ext AS (
+	SELECT se ->> 'statistics_schemaname' statistics_schemaname,
+		   se ->> 'statistics_name' statistics_name,
```

Comment:
Ugh, does anyone know why the tab being 8 spaces here? Asking for a friend.

Reply:
Might be something to do with gofmt using tab with 8 spaces previously: https://github.com/golang/go/issues/19618
Unless someone objects, I'll keep it as is and go with 4 spaces tab here (since my screen is not so wide, 8 spaces is a bit of luxury).